### ISSUE TITLE
[v2] Missing __New(…) meta functions

### ISSUE THREAD

**Issue by Bunker-D** [mark-wiemer/ahkpp#586]
## Issue

Some `__New` meta function are missing from the `ahk2.d.ahk`. For example:
```ahk
class MyHook extends InputHook {
    __New() {
        super.__New("L1")
    }
}
```
results in `__New` being highlighted with the message `Class 'InputHook' might not have member '__New'`.

I guess it was missing because not in the documentation? For the other classes I've checked, `__New` was recognized by the extension if and only if it was explicitly mentioned in the documentation.

## Possible solutions

If I'm not mistaken, every object class has a tacit `__New(…)` method, and its signature and description can be directly copied from its `Call(…)` method, except it returns nothing. So I guess it could be used to generate the missing `__New(…)` methods.

Alternatively, it seems like [`thqby`](https://github.com/thqby)'s [`vscode-autohotkey2-lsp`](https://github.com/thqby/vscode-autohotkey2-lsp) currently include those `__New(…)` methods. At least it does for `InputHook`.

**Comment by mark-wiemer** [comment]
Makes sense, this was probably "lost" when I switched to GroggyOtter's syntax files, I'll see what I can do. Ref #549 for the curious :)

**Comment by mark-wiemer** [comment]
(GitHub UX is being a bit wonky with project links appearing right now, as I recently moved the repo out of `mark-wiemer-org` and into `mark-wiemer`, but didn't move the project. Ref [AHK++ project](https://github.com/orgs/mark-wiemer-org/projects/1?query=sort%3Aupdated-desc+is%3Aopen) for those that want to track priority of this issue)

**Comment by mark-wiemer** [comment]
All this and more coming to 6.5.0! Thanks for reporting this :)

![screenshot of InputHook.__New documentation](https://github.com/user-attachments/assets/767b7642-d83b-4b73-8de8-6710c867e43d)
